starrocks/benchmark.sh

```diff
@@ -1,3 +1,60 @@
 #!/bin/bash
 
 # Install
+wget https://download.starrocks.com/en-US/download/request-download/40/StarRocks-2.3.0-rc03.tar.gz
+tar zxvf StarRocks-2.3.0-rc03.tar.gz
+cd StarRocks-2.3.0-rc03/
+
+# Install dependencies
+yum install -y java-1.8.0-openjdk-devel.x86_64
+
+# Create directory for FE and BE
+IPADDR=`hostname -i`
+export STARROCKS_HOME=`pwd`
+mkdir -p meta storage
+
+# Start Frontend
+echo "meta_dir = ${STARROCKS_HOME}/meta " >> fe/conf/fe.conf
+fe/bin/start_fe.sh --daemon
+
+# Start Backend
+echo "storage_root_path = ${STARROCKS_HOME}/storage" >> be/conf/be.conf
+echo "disable_storage_page_cache = false" >> be/conf/be.conf
```

Comment:
According to the benchmark methodology, if an internal cache is used, it should be flushed before the first "cold" query run. But I don't see it in `run.sh`.

Reply:
I see. We should flush all cache before the query, but right row we don't support flush the internal page cache. Since this feature is on the way, we would confirm the performance after we implement this feature.